examples/external-mysql/values.yaml

```diff
@@ -2,7 +2,8 @@ spire-server:
   dataStore:
     sql:
       databaseType: mysql
-      databaseName: spire
-      host: mysql
+      databaseName: spire-server
+      host: mysql.mysql
       port: 3306
       username: spire
+      password: sp1ff3Test
```

Comment:
Yes thought about that, but also intended here to make examples more straightforward. If there is to much plumbing in examples it also complicates it for our consumers to follow the example and therefore looses its value as an example.

In the end `power comes with great responsibility` and users could copy paste things from many places on the internet or chatgpt. They do need to do their due diligence.

We could however add a note in the README.md to warn them about this hardcoded password. Let me add that at the minumum.

Reply:
It would be possible for the run-tests.sh to include the password hardcoded via --set and have the values file have it commented out with a value of CHANGEME, indicating to the user they should set it to something if they are to deploy it themselves using the values file? Maybe a good middle ground.

Reply:
Smart idea. Done.